css-brochure-form/styles.css

```diff
@@ -0,0 +1,63 @@
+form {
+  width: 50%;
+  margin: auto;
+  margin-left: 6.8in;
+}
+
+p {
+  padding-left: 10px;
+  margin-bottom: 6px;
+  font-family: BrandonGrotesque, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+input {
+  margin: 4px;
+  width: 53%;
+  padding: 10px;
+  border-style: solid;
+  border: 1px solid rgb(226 226 226);
+  border-radius: 5px;
+  font-size: 17px;
+  font-family: AtlasGrotesk, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  flex
+}
+
+h1 {
+  text-align: center;
+  color: rgb(211 89 71);
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: bold;
+  font-weight: 300;
+  font-size: 20px;
+}
+
+select {
+  margin: fixed;
+  width: 458px;
+  height: 47px;
+  background-color: rgb(240 240 240);
+  border-style: solid;
+  border: 2px solid rgb(226 226 226);
+  border-radius: 5px;
+  font-size: 13px;
+}
+
+button {
+  margin: auto;
+  background-color: rgb(219 83 65);
+  font-size: 16px;
+  color: white;
+  text-align: right;
+  margin: 10px 1px;
+  padding: 9.5px 25px;
+  border-style: solid;
+  border: 1px solid rgb(226 226 226);
+  border-radius: 25px;
+  position: fixed;
+  right: 615px;
```

Comment:
Don't do this. `fixed` glues the button to the viewport, it is even worse than `absolute` unless you intend to glue an item to the viewport. If this page had any scroll, that button would never move.